plugin.xml

```diff
@@ -29,6 +29,11 @@
       <author>Benjamin Fontan</author>
    </authors>
    <versions>
+      <version>
+          <num>2.10.5</num>
+          <compatibility>~10.0.0</compatibility>
+          <download_url>https://github.com/pluginsGLPI/order/releases/download/2.10.5/glpi-order-2.10.5.tar.bz2</download_url>
```

Comment:
Non blocking, just wondering if this is needed (we don't do it for every plugin) ?

ping @cedric-anne